libs/island-ui/core/src/lib/Hyphen/Hyphen.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useMemo } from 'react'
+import ReactHtmlParser from 'react-html-parser'
+import Hypher from 'hypher'
+import { is } from './patterns'
```

Comment:
I wonder if we can/should dynamically import this

Reply:
That might be a good idea, I think this is sufficient while we only have one pattern but we should look into having this dynamically imported when/if we add locale support